src/Projection/User/UserFinder.php

```diff
@@ -53,6 +53,19 @@ public function findById($userId)
     }
 
     /**
+     * @param string $emailAddress User's email address
+     * @return bool
+     */
+    public function emailAddressExists($emailAddress)
```

Comment:
but then, I would have to return the whole user instead of just a boolean. It would be more slow and a bit overengineered.